projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_card.scss

```diff
@@ -31,6 +32,14 @@ $_dark-bootstrap-card: extend(
 
 /// Generates a dark indigo card schema.
 /// @type {Map}
+/// @prop {Map} background [igx-color: ('grays', 700)]- The card background color.
 /// @requires extend
 /// @requires $_indigo-card
-$_dark-indigo-card: extend($_indigo-card);
+$_dark-indigo-card: extend(
+    $_indigo-card,
+    (
+        background: (
+            igx-color: ('grays', 700)
```

Comment:
This is not the right background color. The title, subtitle and content colors should also be changed.